server/controllers/chartsController.js

```diff
@@ -5,19 +5,25 @@ import { collections } from '../utils/mongoCollections'
 const TOTALS_STUDY = 'Totals'
 const STUDIES_TO_OMIT = ['files', 'combined']
 
-const studyCountsToPercentage = (studyCount, targetTotal) =>
-  (100 * +studyCount) / targetTotal
+const studyCountsToPercentage = (studyCount, targetTotal) => {
+  if (Number.isNaN(+studyCount) || Number.isNaN(+targetTotal)) {
+    return 0
+  }
+
+  return (+studyCount / +targetTotal) * 100
+}
```

Comment:
🙌 Nice, should we also make sure `targetTotal` is not `0`? 
As written above, if we call this with `studyCountsToPercentage(5, 0)` I believe we'll get `Infinity`
```suggestion
const studyCountsToPercentage = (studyCount, targetTotal) => {
  if (!targetTotal || Number.isNaN(+studyCount) || Number.isNaN(+targetTotal)) {
    return 0
  }

  return (+studyCount / +targetTotal) * 100
}
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/159/commits/bac9a58114af036773a1082131399f1c12425c2d